.env_example

```diff
@@ -12,3 +12,4 @@ MAKER_FEE_ASSET_DATA=
 TAKER_FEE_UNIT_AMOUNT=
 POSTGRES_URI=
 WHITELIST_ALL_TOKENS=
+LIQUIDITY_POOL_REGISTRY_ADDRESS=
```

Comment:
Oh nice. Can you add the ones we forgot :) 
`MESH_IGNORED_ADDRESSES`
`SWAP_IGNORED_ADDRESSES`